Update docs. (#23)
iotaledger/inx-poi: documentation/docs/reference/api_reference.md

# API Reference

> You can find specifications of the INX-POI API by the link below:

Reviewer: ```suggestion
You can find the specifications for the INX-POI API in the following link:
```
